flow/src/org/labkey/flow/script/ScriptXarSource.java

```diff
@@ -94,9 +88,10 @@ public ExperimentArchiveDocument getDocument()
         return _doc;
     }
 
+
     @Override
-    public File getLogFile()
+    public Path getLogFilePath()
     {
-        return _logFile;
+        return _logFile.toPath();

```

Comment:
Could `_logFile` be null?

Reply:
It's initialized in the constructor so we should always have a value.